doc/ApiDocGeneration/templates/matthews/styles/main.js

```diff
@@ -118,9 +133,12 @@ function populateOptions(selector, packageName) {
 
         $(versionselector).change(function () {
             targetVersion = $(this).val()
+            original
```

Comment:
What does this line do?

Reply:
This is added by accident. I will remove it. The code tested in local. There is no good choice to test in prod